ADO.NET/AdoOauthSample/OAuthRefreshSettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "OAuthTokenUrl": "your_token_url",

```

Comment:
The secret is here (along with the other settings) because I wanted to show how we currently do refresh - by getting the url, client secret, and client ID from the json string.